d3/d3.d.ts

```diff
@@ -1224,6 +1269,13 @@ declare module D3 {
                 (angle: (d : any) => number): PieLayout
                 (angle: (d : any, i: number) => number): PieLayout;
             };
+            padAngle:{
+                ():number;
+                (angle:number):PieLayout;
+                (angle:()=>number):PieLayout;
+                (angle: (d : any) => number): PieLayout
+                (angle: (d : any, i: number) => number): PieLayout;
```

Comment:
I think there are some problems in the signature of `padAngle`, `startAngle` and `endAngle`. According to the [D3 source code](https://github.com/mbostock/d3/blob/master/src/layout/pie.js#l16#l18) of this three method, the param `i:number` is needless and should not be passed in. But I am not sure, I have posted a discussion in the google group of D3. wait for response.